config/longrun_configs/amip_target_topo.yml

```diff
@@ -9,11 +9,16 @@ dt_cpl: 100
 dt_rad: "1hours"
 dt_save_state_to_disk: "20days"
 dt_save_to_sol: "10days"
+ode_algo: "ARS343"
 energy_check: false
 hourly_checkpoint: true
 land_albedo_type: "map_temporal"
 mode_name: "amip"
 mono_surface: false
+hyperdiffusion: "ClimaHyperdiffusion"
+vorticity_hyperdiffusion_coefficient: 1.0
+scalar_hyperdiffusion_coefficient: 4.0
```

Comment:
these are also specified in the atmos config: https://github.com/CliMA/ClimaAtmos.jl/blob/main/config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_0M_earth.yml

It's okay to specify them here too (it will overwrite the atmos options), but it does make the coupler config longer/more complex